src/main/java/uk/gov/register/db/EntryMerkleLeafStore.java

```diff
@@ -1,42 +1,41 @@
 package uk.gov.register.db;
 
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.SerializationFeature;
+import uk.gov.objecthash.IntegerValue;
+import uk.gov.objecthash.ListValue;
+import uk.gov.objecthash.ObjectHashable;
+import uk.gov.objecthash.RawValue;
+import uk.gov.objecthash.StringValue;
+import uk.gov.objecthash.TimestampValue;
 import uk.gov.register.core.Entry;
-import uk.gov.register.views.EntryView;
 import uk.gov.verifiablelog.store.MerkleLeafStore;
 
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
 public class EntryMerkleLeafStore implements MerkleLeafStore {
     private final EntryIterator entryIterator;
 
-
     public EntryMerkleLeafStore(EntryIterator entryIterator) {
         this.entryIterator = entryIterator;
     }
-
     @Override
     public byte[] getLeafValue(int i) {
         return bytesFromEntry(entryIterator.findByEntryNumber(i + 1));
```

Comment:
what's the purpose of the `+ 1` here, is it something to do with things being 0 indexed?

Reply:
Entry number starts at 1, but MerkleLeafStore implementations should be zero indexed.